application/src/main/resources/ecc.yml

```diff
@@ -91,9 +91,10 @@ connection:
     ## Specify the interval until the next connection to a node
     ## Unit can be SECONDS, MINUTES, HOURS, DAYS
     ##
-    connectionDelay:
-      time: 45
-      unit: MINUTES
+  connectionDelay:
```

Comment:
I think you need to rebase your change, I changed this to be on the same level of native connection configuration because of the scheduler changes

Reply:
Rebased now